apps/src/sites/studio/pages/init/loadGamelab.js

```diff
@@ -8,6 +8,10 @@ export default function loadGamelab(options) {
   options.skinsModule = skins;
   var gamelab = new GameLab();
 
+  if (IN_UNIT_TEST) {
+    window.Gamelab = gamelab;
+  }
```

Comment:
Caley did this in conjunction with me. I wasn't sure how I felt about the IN_UNIT_TEST approach. Hadn't considered __TestInterface. That might be better. Would be interested if @islemaster has an opinion.

Worth noting that our other apps (such as AppLab) are just always on window